app/models/bet_option.rb

```diff
@@ -2,6 +2,12 @@ class BetOption < ApplicationRecord
   has_many :users, through: :user_bets
   has_many :user_bets
 
+  def winner_total
```

Comment:
I dont think this should be winner_total. Not all bets are winners,

Also, instance methods should be placed below class methods (scopes)